client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/GeneratorProcessor.java

```diff
@@ -61,6 +64,10 @@ void additionalBean(
             Capabilities capabilities,
             BuildProducer<AdditionalBeanBuildItem> producer) {
 
+        if (!isClassPresentAtRuntime(ABSTRACT_TOKEN_PRODUCER)) {
```

Comment:
the oidc client does not declare any capability, I need to use this as a fallback

Reply:
https://quarkusio.zulipchat.com/#narrow/channel/187038-dev/topic/Default.20Capability.20For.20extension